OPEN_PROBLEMS/DECENTRALIZED_DATA_DELIVERY_MARKETS.md

```diff
@@ -0,0 +1,511 @@
+
+#### Properties desired
+
+*   The exchanges of value MUST be verifiable and correct
+    *   Fairness: 
+        *   The payment MUST be fulfilled if the SLA is fulfilled
+        *   The payment for bandwidth/latency in token SHOULD match what has been agreed and fulfilled
+    *   Verifiability:
+        *   Both parties MUST be capable of verifying that the exchange was performed correctly
+        *   Bonus: Anyone SHOULD be able to verify that the exchange was performed correctly
+*   SHOULD be Trustless
+    *   Ideally, the network can perform the operations without having to leverage Trust
+    *   Nevertheless in many designs it might be needed to leverage reputation as a trust vehicle to guarantee high quality of service and/or reliability.
+
+#### How is Data Delivery Metering different from Fair Exchange
+
+The field of Fair Exchange overlaps in part with Service Metering. Some notable differences are:
+
+*   Fair Exchange can be used for non-digital goods also
+*   Fair Exchange is traditionally between 2 parties
+*   Metering should be available to properly measure the service quality and service provided by one or more parties (e.g. streaming from multiple endpoints)
+
+In review, the Metering & Fair Exchange fields end up contributing to each other and it is likely that the intersection of both is needed for a sound solution. 
+
+#### Known challenges
+
+*   Fairness
+  *   Making it Trustless
+    *   How to verify that the provider has the ability to delivery the file (e.g. stored locally or that can pull it from the network)
+    *   How to verify that the file being transferred is the one requested, before a large amount of resources has been invested (i.e., before the entirety of the file has been transmitted)
+    *   How to verify that the client has received the file
+  *   How to verify and reward that SLA were guaranteed?
+  *   How to avoid collusion when adding a third-party (e.g. Referee)?
+  *   How to avoid grieving attacks (make one party pay for fees)?
+  *   How to avoid a malicious actor causing un-rewarded work, hence wasting the others’ resources?
+*   Experience
+  *   How to make the transfers start instantaneously?
+  *   How to support others (e.g. Content Publishers) paying for the usage?
+  *    How to make it private (e.g. that others don’t know who is requesting what)?
+*   Performance
+  *   How to overcome the send-and-halt in order to max bandwidth throughput 
+      *   How to support multipath (i.e. fetching from multiple sources)
+
+### State-of-the-art
+
+When it comes to the state of the art, we found that the existing solutions fall within one of the following categories:
+
+#### Pay-per-packet
+
+Pay-per-packet solutions offer granular control over what gets paid, enabling the option to verify if the SLA is being fulfilled. However, their setup cost and latency overhead puts them behind on being able to max out the available underlay resources.
```

Comment:
```suggestion
Pay-per-packet solutions offer granular control over what gets paid, enabling the option to verify if the SLA is being fulfilled. However, their setup cost and latency overhead put them at a disadvantage in terms of maximizing utilization of the underlying resources.
```
I _think_ you meant underlying instead of underlay but revert if not.